docroot/themes/humsci/su_humsci_theme/patterns/molecules/featured-card/featured-card.html.twig

```diff
@@ -1,20 +1,38 @@
-{% set attributes = attributes.addClass('card') %}
+{% set attributes = attributes.addClass('vertical-card') %}
 
 <div{{ attributes }}>
 
-<div class="featured-card__container">
-  <div class="featured-card__img">
-  {{ image }}
-  <div class="date-stacked">
-    <div class="date-month">{{ month }}</div>
-    <div class="date-day">{{ day }}</div>
+  <div class="featured-card__container">
+    {% if image %}
```

Comment:
This if statement could cause problems. It is wrapping the opening div tag for an image but there is no if statement for the closing tag. This could leave hanging </div>s

Reply:
@josephgknox This needs to be fixed.

Reply:
Thanks, @sherakama! I'm combining some work from a few tickets, so will be updating this on the ARCH-72 branch (which you also commented on).